frontend/src/services/apiService.ts

```diff
@@ -1,8 +1,18 @@
 /* Interactions with the FastAPI backend */
 import axios from 'axios';
 
+interface CheckSessionIdResponse {
+  found: boolean;
+}
+
+interface FreshSessionIdResponse {
+  session_id: string;
+}
+
 const apiService = {
-  sendQuery: async (query: string) => axios.post('http://localhost:8000/query/', { user_input: query })
+  sendQuery: async (query: string, sessionId: string) => axios.post('http://localhost:8000/query/', { user_input: query, session_id: sessionId }),
```

Comment:
preferably we should move the `localhost:8000` into ENV variable with a default.

Also we might think of using some fastapi library for typescript, generate schemas and have all that juicy type safety by default ;)

Reply:
sendQuery was already in the code. I dont really want to use to many ENV variables except for secrets as it is extra manual work for everyone. If everyone agrees on it, then i don't mind though.